### PR TITLE
RSSchool code-jam: HTML, CSS & Git Basics

### PR DESCRIPTION
**Pull request by Vladlis**
Task: https://github.com/rolling-scopes-school/tasks/blob/2018-Q3/tasks/codejam-cv.md

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+<head>
+  <meta charset="UTF-8">
+  <title>CV</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <header>
+    <h1>Vladislav Lisetskii</h1>
+    <hr>
```
Comment (sonic-interlock) [inline index.html:13]: it is not an outdated tag, to be honest, but it's usage is kind of discouraged :)
please use border-top or border-bottom css rules instead

File: index.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        <ul>
+          <li>Position: Fullstack developer (Java/Javascript)</li>
+          <li>Age: 25</li>
+          <li>City: Moscow</li>
+        </ul>
+      </div>
+      <div class="contacts">
+        <ul>
+          <li>E-mail: <a href="mailto:vladlis54@gmail.com">vladlis54@gmail.com</a></li>
+          <li>Phone: <a href="phoneto:88005553535">8 (800) 555-35-35</a></li>
```
Comment (sonic-interlock) [inline index.html:25]: >phoneto:
I believe you mean `tel:` or does your phone support phoneto as well? :thinking:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+          <li>Github: <a href="https://github.com/Vladlis">Vladlis</a></li>
+        </ul>
+      </div>
+    </div>
+    <img src="assets/photo.jpeg" alt="photo" class="photo">
+    <hr>
+  </header>
+  <section class="summary">
+    <h2>Summary</h2>
+    <span>I am a Java developer who became tired of constant enterprise development hell, so I have mastered frontend
```
Comment (sonic-interlock) [inline index.html:35]: apparently that should be a paragraph
use span when there is no more suitable tag for some inline element
use div when there is no better tag for a block element
and when you have text in a block element it is a paragraph

File: index.html [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    <h2>English courses</h2>
+    <h4>English school</h4>
+    <span>Got B2 FCE certificate as a result.</span>
+    <hr>
+  </section>
+  <footer class="footer">
+    <span>Have a nice day :)</span>
+  </footer>
+</body>
+</html>
```
Comment (sonic-interlock) [inline index.html:76]: not like I care too much about empty lines, anyway it's a general guideline to have them
I guess you may just configure your editor to insert them automatically

**Review by sonic-interlock — COMMENTED** [review]
100 points, all the requirements are accomplished :)

File: index.html (hunk 1 above)
Comment (Vladlis) [reply index.html:13]: ok, thanks

File: index.html (hunk 2 above)
Comment (Vladlis) [reply index.html:25]: sure, it has to be `tel:`   ))

File: index.html (hunk 3 above)
Comment (Vladlis) [reply index.html:35]: thanks

**Pull request merged**